docs/tutorials/serving/rest_simple.ipynb

```diff
@@ -355,7 +355,12 @@
       },
       "outputs": [],
       "source": [
-        "!{SUDO_IF_NEEDED} apt-get install tensorflow-model-server"
+        "#!{SUDO_IF_NEEDED} apt-get install tensorflow-model-server #installing Tensorflow Model server 2.8 instead of latest version\n",
```

Comment:
Please mark this block clearly as a TODO to rollback this change. For example:

"# TODO: Use the latest model server version when colab supports it.\n",
"#!{SUDO_IF_NEEDED} apt-get install tensorflow-model-server\n",
"# We need to install Tensorflow Model server 2.8 instead of latest version\n",